base/all-all/sbardef.lmp

```diff
@@ -123,7 +301,117 @@
               "type": 1,
               "param": 0,
               "maxlength": 3,
-              "conditions": null,
+              "conditions":
+              [
+                {
+                  "condition": 22,
+                  "param": 0
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "percent":
+            {
+              "x": 235,
+              "y": 3,
+              "alignment": 2,
+              "font": "BigRed",
+              "tranmap": null,
+              "translation": "CRRED",
+              "type": 1,
+              "param": 0,
+              "maxlength": 3,
+              "conditions":
+              [
+                {
+                  "condition": 21,
+                  "param": 0
+                },
+                {
+                  "condition": 22,
+                  "param": 1
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "percent":
+            {
+              "x": 235,
+              "y": 3,
+              "alignment": 2,
+              "font": "BigRed",
+              "tranmap": null,
+              "translation": "CRGOLD",
+              "type": 1,
+              "param": 0,
+              "maxlength": 3,
+              "conditions":
+              [
+                {
+                  "condition": 21,
+                  "param": 25
+                },
+                {
+                  "condition": 22,
+                  "param": 1
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "percent":
+            {
+              "x": 235,
+              "y": 3,
+              "alignment": 2,
+              "font": "BigRed",
+              "tranmap": null,
+              "translation": "CRGREEN",
+              "type": 1,
+              "param": 0,
+              "maxlength": 3,
+              "conditions":
+              [
+                {
+                  "condition": 21,
+                  "param": 50
+                },
+                {
+                  "condition": 22,
+                  "param": 1
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "percent":
+            {
+              "x": 235,
+              "y": 3,
+              "alignment": 2,
+              "font": "BigRed",
+              "tranmap": null,
+              "translation": "CRBLUE",
```

Comment:
We use `CRBLUE2` for the BigRed font, please.

Reply:
Fixed in 5e954e2